main.js

```diff
@@ -0,0 +1,298 @@
+
+// variables to retrieve cell info
+
+let c0 = document.getElementById('cell-0')
+let c1 = document.getElementById('cell-1')
+let c2 = document.getElementById('cell-2')
+let c3 = document.getElementById('cell-3')
+let c4 = document.getElementById('cell-4')
+let c5 = document.getElementById('cell-5')
+let c6 = document.getElementById('cell-6')
+let c7 = document.getElementById('cell-7')
+let c8 = document.getElementById('cell-8')
+
+// other global variables
+
+let cells = document.querySelectorAll("[id^='cell']")
+let playerStatus = document.getElementById("status-bar")
+
+// -----------------------------------------------------------------------------------------------
+// functions
+// function to check whether any winning combination has been achieved during play
+
+function gameWin() {
+  let index = 0
+
+  // loop to iterate through winning array combinations
+
+  while (index < 8) {
+
+    // winning array combinations
+
+    let winningArrays = [
+
+      [c0.innerHTML, c1.innerHTML, c2.innerHTML],
+      [c0.innerHTML, c3.innerHTML, c6.innerHTML],
+      [c0.innerHTML, c4.innerHTML, c8.innerHTML],
+      [c3.innerHTML, c4.innerHTML, c5.innerHTML],
+      [c6.innerHTML, c7.innerHTML, c8.innerHTML],
+      [c1.innerHTML, c4.innerHTML, c7.innerHTML],
+      [c2.innerHTML, c5.innerHTML, c8.innerHTML],
+      [c2.innerHTML, c4.innerHTML, c6.innerHTML]
+
+    ]
+    // function to filter arrays for X's and O's
+
+    checkedMoves = winningArrays[index].filter(function (value) {
+      return value === currentPlayer.symbol
+    })
+    index++
+
+    // check that returns the first array to contain 3 X's or O's
+
+    if (checkedMoves.length === 3) {
+      console.log(index) // I'm not sure how to retrieve (return) this index, but I think I'll need it in order to know which combination was reached so that I can draw a line through the appropriate cells or change their background color.
+      return checkedMoves
+    }
+  }
+}
+
+// function to delay game (i.e. page) reset
+
+function delayEnding() {
+  window.setTimeout(endGame, 4 * 1000)
+}
+
+// function to reset board (i.e. page)
+
+function endGame() {
+  document.location.reload();
+  clearInterval(interval);
+}
+
+// draw function checks if board has been completely filled
+
+function tieGame() {
+
+  let grid =
+    [c0.innerHTML, c1.innerHTML, c2.innerHTML, c3.innerHTML, c4.innerHTML, c5.innerHTML, c6.innerHTML, c7.innerHTML, c8.innerHTML]
+
+  fullGrid = grid.filter(function (value) {
+    return value !== ''
+  })
+  if (fullGrid.length === 9) {
+    return fullGrid
+  }
+}
+
+// function to accept player names, insert player into status area, disable start button and start timer
+
+function startGame2() {
+  playerX.name = xName.value
+  playerO.name = oName.value
+  currentPlayer = playerX
+  playerStatus.innerHTML = currentPlayer.name + "'s Turn";
+  event.target.disabled = true
+  startButton1.disabled = true
+  xName.value = ""
+  oName.value = ""
```

Comment:
Here you can do something like:
```js
xName.style.display = "none";
oName.style.display = "none";
```
and this will remove the inputs from the page after you click start